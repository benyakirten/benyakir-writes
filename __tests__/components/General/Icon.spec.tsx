import * as React from 'react'

import { render, cleanup } from '@TestUtils'

import { Icon } from '@Gen'
import { allIcons } from '@TestProps'

describe('Icon component', () => {
  afterEach(cleanup)

  it('should render properly', () => {
    expect(() => render(<Icon icon={allIcons[0]} />)).not.toThrow()
  })
})
