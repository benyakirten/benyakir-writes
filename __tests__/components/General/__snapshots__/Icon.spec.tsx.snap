// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon component should render properly 1`] = `
<div
  className="sc-jRQBWg hVwmOY"
  height="3rem"
>
  <div />
  <img
    alt="icon"
    className="sc-gKclnd jbCegM"
    height="3rem"
    src="/static/e986765a838c4faa272a607d98fb0088/icon.png"
  />
  <span
    className="sc-iCfMLu tnmtf"
  >
    icon
  </span>
</div>
`;
